jitbuilder/optimizer/JBOptimizer.cpp

```diff
@@ -117,11 +117,12 @@ static const OptimizationStrategy JBwarmStrategyOpts[] =
    { OMR::treeSimplification                                                       },
    { OMR::trivialDeadTreeRemoval,                    OMR::IfEnabled                },
 
-   { OMR::inductionVariableAnalysis,                 OMR::IfLoops                  },
-   { OMR::loopCanonicalization,                      OMR::IfLoops                  },
+   { OMR::basicBlockOrdering                                                       }, // clean up block order going into canonicalizer
+   { OMR::loopCanonicalization,                      OMR::IfLoops                  }, // canonicalizer first, otherwise indvar data gets messed up
```

Comment:
I read that as one continuous sentence with two "canonicalizer"s in a row.  If that's what you intended then please remove the first one (or second one :-)  ).  If that's not what you intended then please add a period at the end of the first comment.